analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -210,5 +211,22 @@ private ProgramState ResetFieldConstraints(Func<IFieldSymbol, bool> predicate)
             }
             return state;
         }
+
+        [Conditional("DEBUG")]
+        public void CheckStateConsistency()
```

Comment:
This needs something likee `[ExcludeFromCoverage]`, or how is the attribute called.

Reply:
AltCover needed to be instrumented to respect the attribute. I would like to also add the `Conditional` attribute to the attribute filter list. What do you think?

Reply:
I wouldn't mix them. The explicit `ExcludeFromCoverage` makes it clear about what it does. While `Conditional` could be used for other stuff too (even when we don't do it for now)